java/sage/Wizard.java

```diff
@@ -3115,13 +3116,10 @@ public boolean addSeriesInfoPublic(int legacySeriesID, String title, String netw
     return addSeriesInfo(legacySeriesID, title, network, description, history, premiereDate, finaleDate, airDOW, airHrMin, imageURL, people, characters) != null;
   }
   public SeriesInfo addSeriesInfo(int legacySeriesID, int showcardID, String title, String network, String description, String history, String premiereDate,
-      String finaleDate, String airDOW, String airHrMin, String[] people, String[] characters, byte[][] imageURLs)
+      String finaleDate, String airDOW, String airHrMin, Person[] people, String[] characters, byte[][] imageURLs)
```

Comment:
Please don't remove any of these 'add' methods...it'll break the other EPG plugin I'm using currently (which of course I'm going to ditch in a few months). Just overload it with another one. :)